tensorflow/python/keras/applications/mobilenet.py

```diff
@@ -56,9 +56,10 @@
 |  1.0 MobileNet-128  |    64.4 %    |        529        |     4.2     |
 ------------------------------------------------------------------------
 
-Reference paper:
-  - [MobileNets: Efficient Convolutional Neural Networks for
-     Mobile Vision Applications](https://arxiv.org/abs/1704.04861)
+Reference:
+  - [MobileNets: Efficient Convolutional Neural Networks
+     for Mobile Vision Applications](
+      https://arxiv.org/abs/1704.04861)
```

Comment:
No need for this

Reply:
Do you mean splitting the link into 3 lines? I just copied it from line 98 below, but I can make it two lines again. (Not sure why it's like this below)